src/sagas/console/index.js

```diff
@@ -86,6 +86,50 @@ export function* downloadVmConsole (action) {
   }
 }
 
+/**
+ * Legacy options were saved in browser's local storage.
+ * The UI for setting theose options was removed in previous versions.
+ * However there is still a (small) chance that the data is still there.
+ * Note that legacy options are per VM and therefore should overwrite global defaults.
+ */
+function* getLegacyOptions ({ vmId }) {
```

Comment:
Man that is a VERY small chance.  The switches for console config hasn't existed since before the VM Detail's screen was redesigned at least 2 years ago.

Reply:
Yes, I agree. However removing the legacy mechanism was not in scope of this PR. Beside that ....the code for reading the values is still in our code base and most likely working:)